src/Aspire.Hosting/OpenAI/OpenAIBuilderExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Publishing;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding OpenAI resources to the application model.
+/// </summary>
+public static class OpenAIBuilderExtensions
+{
+    /// <summary>
+    /// Adds an OpenAI service to the application model.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{OpenAIResource}"/>.</returns>
+    public static IResourceBuilder<OpenAIResource> AddOpenAI(this IDistributedApplicationBuilder builder, string name)
```

Comment:
This doesn't make sense to me. Let's say I'm building an app that uses Open AI (not Azure Open AI) and I want to deploy it on ACA via AZD. This won't work because the manifest will contain an `openai.v0` resource which AZD doesn't know how to handle.

What we'll probably need to do here is have this resource type emit a `connectionString` field which contains the Open AI connection string. But for security reasons it would reference a secretstore/secret which in turn would use an `input` field to capture this detail during deployment.

We can get this in to main and I can help you work through this though since the secrets stuff is pretty new and not baked into AZD (yet).

Reply:
> This won't work because the manifest will contain an openai.v0 resource which AZD doesn't know how to handle.

I thought the idea of the manifest was to describe all it could, and the consumers to convert to what they can use. In this case would do nothing for `openai.v0`.

Should I remove this line from the manifest at least?